src/worker/background.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
this file should land in the `xcode` folder, next to the manifest.json for Safari extension

Reply:
I am not sure the reasoning here, so I would need some deeper explanation. It is a part of codebase, and even if it is not required for Chrome, we only deliver it for Safari for now. Keeping codebase in one place is much clearer, if we would change for example name of the entry point for the background process. Also, it might be a case that in future if Firefox will be supported by this extension, it will also require using HTML over the service worker (It is my assumption, I don't know exactly how Mozilla plans to support manifest v3).

Reply:
this code is specific for Safari extension. No needed for Chromium or Firefox at this moment. It is also tiny so it does not matter. The point would be to not ship it with Chromium extension as there is no reason to do so. So in that way moving it to Safari specific folder (same as Safari specific manifest) makes project cleaner. But as I've already said, it does not matter :)